### DISCUSSION TITLE
TuneIn AAC Streams not longer working / no sound

### DISCUSSION THREAD

**Discussion by Stumpi73** [music-assistant/support#2089] [Q&A] [answered]
Hi,
since one o f the last versions (dont know exactly wich version) AAC is not longer playing sound in TuneIn Radiostations.
Specially the Radios 80s80s/90s90s. MP3 stations are working normal.
Can anyone confirm this?

**Comment by marcelveldt** [comment, marked as answer]
If you had followed the conversations on discord and existing issues, you would have known that this is a known issue being worked on. Next update will fix it
